### ISSUE TITLE
Make the website a PWA

### ISSUE THREAD

**Issue by Abdulkadir98** [codezoned/codezoned.com#24]
I could take it up or work with someone who's already working on it

**Comment by ionicc** [comment]
Yes, We had plans of revamping the website with a PWA approach. It'll be great if you could talk to us about your approach on the forum and join us on discord.

**Comment by Abdulkadir98** [comment]
Sure, will do!

**Comment by arjunmahishi** [comment]
I think something like Gatsby.js should be used to build websites like this. I stumbled onto it recently. If the website is being rebuilt, I'd love to contribute too.

**Comment by Ryuno-Ki** [comment]
Where's the Discord invitation link?

**Comment by Ryuno-Ki** [comment]
As alternative to Gatsby (which requires React), consider Eleventy (allows you to pick different template languages).
Be aware, that this raises the complexity, as you have to deal with a build process then.

**Comment by Ryuno-Ki** [comment]
Found the Discord link on the website m(
